### ISSUE TITLE
Error in $operation.d.ts when upgrading from typescript 2.8.x to 2.9.x

### ISSUE THREAD

**Issue by nihaux** [types/npm-ramda#413]
Hello, when I upgrade from 2.8 to 2.9 I get this error:  
```
node_modules/@types/ramda/src/$operation.d.ts
(15,54): Type 'keyof T' does not satisfy the constraint 'string'.
  Type 'string | number | symbol' is not assignable to type 'string'.
    Type 'number' is not assignable to type 'string'.
```

the typing in question
```
export type Omit<T, K extends string> = Pick<T, Diff<keyof T, K>>;
```
Is it an incompatibility or am I missing sth ?

**Comment by ikatyang** [comment]
It seems you're using an old version, this should have been fixed by #404.

https://github.com/types/npm-ramda/blob/9529aa3c8ff70ff84afcbc0be83443c00f30ea90/src/%24operation.d.ts#L8

**Comment by nihaux** [comment]
@ikatyang thx for the quick answer !  
I had indeed an old version.  
It was due to yarn having an old version of the package in cache.  
If I may ask, why isn't there a release/version bump process with this package ? Wouldn't it avoir these kind of problem ?
